Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -1,18 +1,55 @@
 """This contains the plotting and geo data management methods for the COVIDcast signals."""
 
 import os
+from datetime import date
+from typing import Tuple
 
 import geopandas as gpd
+import numpy as np
 import pandas as pd
-
 from covidcast import covidcast
+from matplotlib import pyplot as plt
```

Comment:
probably needs to go in setup.py/requirements.txt

Reply:
added